src/controller/CHIPDeviceController.h

```diff
@@ -37,6 +37,8 @@
 #include <transport/SecureSessionMgr.h>
 #include <transport/raw/UDP.h>
 
+#include <string>
```

Comment:
what do you need `<string>` for in this header?

Reply:
This is added before because we used string to store wifi credentials before, removed since this PR no longer uses std::string